### PR TITLE
Feature: Add a question about case styles

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1414,33 +1413,55 @@ const generalCSQuiz = [
...
+    Answer:
+      "Every word is lower case in grouped together words and separated by hyphens.",
+    Distractor1:
+      "Every word has the first letter capitalized in grouped together words.",
+    Distractor2:
+      "Every word is lower case in grouped together words and separated by underscores.",
+    Distractor3:
+      "Every word has the first letter capitalized except the first in grouped together words.",
+    Explanation:
+      "Kebab case has all words are lowercase, and each word gets separated by a dash.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1462]: I think this questions would work better to display an example

```suggestion
    Question: "Which of the following examples shows the correct syntax for kebab case?",
    Answer: "number-of-donuts",
    Distractor1: "number_of_donuts",
    Distractor2: "numberOfDonuts",
    Distractor3: "number*of*donuts",
    Explanation: "Kebab case separates each lowercase word by dashes.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/general-cs-quiz.ts (hunk 1 above)
Comment (a2937) [reply src/data/general-cs-quiz.ts]: Thank you for your suggestion. It definitely is a lot better to use an example in this case.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**